pkg/apis/operator/v1alpha1/tektonresult_types.go

```diff
@@ -97,6 +97,8 @@ type ResultsAPIProperties struct {
 	LoggingPluginStaticLabels           string `json:"logging_plugin_static_labels,omitempty"`
 	LoggingPluginCACert                 string `json:"logging_plugin_ca_cert,omitempty"`
 	LoggingPluginForwarderDelayDuration *uint  `json:"logging_plugin_forwarder_delay_duration,omitempty"`
+	LoggingPluginQueryLimit             *uint  `json:"logging_plugin_query_limit,omitempty"`
+	LoggingPluginQueryParams            string `json:"logging_plugin_query_params,omitempty"`
```

Comment:
where are we using these fields in Operator :thinking:

Reply:
So that someone can configure these fields. 
Options doesn't work with Results Configmap. If someone sets that fields, he has to set all fields or it gets zeroed. We already discuss this and will change format of config to `yaml` from `env` file.

Reply:
Oh okok thanks for the clarification

Just one more query

> Options doesn't work with Results Configmap

why Options won't work for results

Reply:
So we use `env` file format in Results API's Server Configmap (watcher and retention agent uses the format used in other tekton projects).

Now, all configs in `env` file format are part of a var `config` under `data`.  When user change fields, they need to change `config`  var. Options will change all other fields also under config because it assumes `config` var have changed.

Reply:
Refer here: https://github.com/tektoncd/results/blob/ed99c478dc496bde87e3a4fc9a0e418872db6991/test/e2e/gcs-emulator.yaml#L70